src/main/java/org/opengis/cite/gmljpx20/core/CoreTests.java

```diff
@@ -396,7 +396,7 @@ public void containsGmlRangeTypeDataRecordUom() {
             XMLBox auxXmlBox = (XMLBox) xmlBox;
             Document doc = docBuilder.parse(new InputSource(new StringReader(auxXmlBox.xmldata.trim())));
             // Note: Just check doc element for allowed coverage types?
-            Boolean hasGmlCovDataRecordsElems = (Boolean) XMLUtils.evaluateXPath(doc, "//swe:DataRecords:*",
+            Boolean hasGmlCovDataRecordsElems = (Boolean) XMLUtils.evaluateXPath(doc, "//*[local-name()=’swe:DataRecords’]",
```

Comment:
’swe:DataRecords’ is not a valid local-name. `//swe:DataRecords` should be fine. Namespace Binding for 'swe' must be added.